orchagent/portsorch.cpp

```diff
@@ -456,114 +444,149 @@ bool PortsOrch::initializePort(Port &p)
     return true;
 }
 
-bool PortsOrch::setupVlan(sai_vlan_id_t vlan_id, sai_object_id_t port_id, sai_object_id_t &vlan_member_id)
+bool PortsOrch::addHostIntfs(sai_object_id_t id, string alias, sai_object_id_t &host_intfs_id)
```

Comment:
i think this id can be either port id or rif id, so leave it here for general purposes.